packages/aragon-wrapper/src/index.test.js

```diff
@@ -1468,3 +1468,17 @@ test.only('should set forwarded actions', async (t) => {
     }])
   })
 })
+
+test('should init the appMetadata correctly', async (t) => {
```

Comment:
Please add a test here to ensure a "to" param can be passed and read from at this layer, and the wrapper implementation behaves as expected in other ways. There should be more than just an initialization test. Make sure edge cases behave as expected, etc.

Reply:
Done.